charts/ocis/values.yaml

```diff
@@ -21,170 +47,397 @@ image:
+  # type: array
+  # items:
+  #   type: string
+  #   required: true
+  # @schema
   # -- Nodes of the cache to use.
   nodes:
     - "{{ .appNameNats }}:9233"
 
 store:
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   # -- Configure the store type.
   # Can be set to "nats-js-kv" or "redis-sentinel". There is also the non-recommended option "memory".
   # The address of NATS / Redis Sentinel node(s) needs to be set to `cache.nodes`.
   type: nats-js-kv
+  # @schema
+  # type: array
+  # items:
+  #   type: string
+  #   required: true
+  # @schema
   # -- Nodes of the store to use.
   nodes:
     - "{{ .appNameNats }}:9233"
 
 registry:
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   # -- Configure the service registry type.
   # Can be set to "nats-js-kv".
   # The address of NATS node(s) needs to be set to `cache.nodes`.
   type: nats-js-kv
+  # @schema
+  # type: array
+  # items:
+  #   type: string
+  #   required: true
+  # @schema
   # -- Nodes of the service registry to use.
   nodes:
     - "{{ .appNameNats }}:9233"
 
 messagingSystem:
   external:
+    # @schema
+    # type: boolean
+    # required: true
+    # @schema
     # -- Use an external NATS messaging system instead of the internal one.
     # Recommended for all production instances.
     # Needs to be used if HighAvailability is needed.
     # Needs to be used if oCIS shall be used by more than a 2-digit user count.
     enabled: false
+    # @schema
+    # type: [string, null]
+    # required: false
+    # @schema
     # -- Endpoint of the messaging system.
-    endpoint: "nats.ocis-nats.svc.cluster.local:4222"
+    endpoint:
+    # @schema
+    # type: [string, null]
+    # required: false
+    # @schema
     # -- Cluster name to use with the messaging system.
-    cluster: "ocis-cluster"
+    cluster:
     tls:
+      # @schema
+      # type: boolean
+      # required: true
+      # @schema
       # -- Enables TLS encrypted communication with the messaging system.
       # Recommended for production installations.
       enabled: true
+      # @schema
+      # type: boolean
+      # required: true
+      # @schema
       # -- Set only to false, if the certificate of your messaging system service is not trusted.
       # If set to false, you need to put the CA cert of the messaging system server into the secret referenced by "messagingSystemCaRef"
       certTrusted: true
+      # @schema
+      # type: boolean
+      # required: true
+      # @schema
       # -- Disables SSL certificate checking for connections to the messaging system server.
       # -- For self signed certificates, consider to put the CA cert of the messaging system secure server into the secret referenced by "messagingSystemCaRef"
       # Not recommended for production installations.
       insecure: false
 
+# @schema
+# type: array
+# required: true
+# items:
+#   type: object
+#   additionalProperties: false
+#   properties:
+#     ip:
+#       type: string
```

Comment:
We could use https://json-schema.org/understanding-json-schema/reference/string#ip-addresses here.

Reply:
I looked into it but we can only specify one format. So we would need to choose between v4 and v6 IPs.... -> :hankey: 

Maybe I overlooked something?